yaml/csr.yaml.m4

```diff
@@ -117,6 +123,93 @@
       msb: 31
       lsb: 0
 
+ifdef([[[ZICNTR]]], [[[
```

Comment:
Is it better to use C-like booleans like `== 0` for false and `>= 0` for true, instead of ifdef?
That would force a user to always explicitly specify all options/params so that parts aren't accidentally omitted, and it would obviate the handy "ifdefeval" macro above.

Reply:
That is much safer, and eliminates a worry that preprocessing scripts will silently make a wrong decision. I pushed a commit to fix this.